docs/illumos-and-smartos-basics/glossary.md

```diff
@@ -0,0 +1,367 @@
+Solaris term.
+
+## Head node
+
+This is the physical server/node that controls all the other physical and
+virtual servers. It controls the provisioning and maintenance of
+[SmartMachines](glossary.md#smartmachine) and Virtual Machines.
+
+## Hypervisor
+
+"A hypervisor, also called virtual machine manager (VMM), is one of many
+hardware virtualization techniques that allow multiple operating systems to run
+concurrently on a host computer." - [wikipedia](https://en.wikipedia.org/wiki/Hypervisor)
+
+## IP-KVM
+
+Keyboard/video/mouse.
+
+## IaaS
+
+"Cloud infrastructure services, also known as Infrastructure as a Service
+(IaaS), deliver computer infrastructure – typically a platform virtualization
+environment – as a service. Rather than purchasing servers, software,
+data-center space or network equipment, clients instead buy those resources as
+a fully outsourced service. Suppliers typically bill such services on a utility
+computing basis; the amount of resources consumed (and therefore the cost) will
+typically reflect the level of activity. IaaS evolved from virtual private
+server offerings." - [wikipedia](https://en.wikipedia.org/wiki/IaaS)
+
+## illumos
+
+An operating system derived from Sun Microsystems' [OpenSolaris](https://en.wikipedia.org/wiki/Opensolaris).
+
+## Kernel
+
+The main component of a computer operating system, which manages system
+resources and communications between those resources and applications running
+on the system.
+
+## KVM
+
+Kernel virtual machine: a virtual machine implementation using the operating
+system's kernel. - [wikipedia](https://en.wikipedia.org/wiki/Kernel-based_Virtual_Machine)
+
+## LAN
+
+Local Area Network.
+
+## Latency
+
+The time an operation takes to complete; "a measure of customer pain". Lower =
+better.
+
+## Logical Network
+
+A mechanism that SmartDataCenter uses to associate network segments with
+physical network adapters on the head node and on compute nodes.
+
+## Managed hosting
+
+"A type of Internet hosting in which the client leases an entire server not
+shared with anyone." - [wikipedia](https://en.wikipedia.org/wiki/Managed_hosting)
+
+## MAPI
+
+The Cloud Management API provides low-level access to SmartDataCenter. Every
+head node runs an instance of MAPI.
+
+## NAS
+
+Network-attached storage.
+
+## Network virtualization
+
+A set of features in OpenSolaris, Illumos and SmartOS which provides an internal
+network virtualization and quality of service scenario. -
+[wikipedia](https://en.wikipedia.org/wiki/OpenSolaris_Network_Virtualization_and_Resource_Control)
+
+## NIC
+
+Network Interface Card (or Controller).
+
+## NIC tags
+
+NIC tags (network interface card/controller tags) are the physical network label
+(a name not a number) that is associated with an active network interface port.
+Each active network interface port will have a MAC (media access control) from
+the NIC vendor and SDC NIC_tag.
+
+## NoSQL
```

Comment:
I'm not sure this entry is useful

Reply:
Since we are not sure if is useful, I removed the term. I think that is better starting only having what we think that is useful.